applications/ContactStructuralMechanicsApplication/custom_conditions/ALM_mortar_contact_condition.h

```diff
@@ -392,60 +396,40 @@ class AugmentedLagrangianMethodMortarContactCondition: public Condition
     ///@{
     
    /**
-    * This struct is used in the component wise calculation only
-    * is defined here and is used to declare a member variable in the component wise condition
-    * private pointers can only be accessed by means of set and get functions
-    * this allows to set and not copy the local system variables
+    * This struct is used to store the flags and components of the local system
     */
-    struct LocalSystemComponents
+    struct LocalSystem
     {
     private:
-            //for calculation local system with compacted LHS and RHS
-            MatrixType *mpLeftHandSideMatrix;
-            VectorType *mpRightHandSideVector;
+        // For calculation local system with compacted LHS and RHS
+        MatrixType *mpLeftHandSideMatrix;
```

Comment:
why do you need to store a raw_pointer to the LeftHandSide matrix? what is it for?

Reply:
Just to keep the structure what it was done previously